Containerfile

```diff
@@ -14,7 +14,8 @@ FROM $PYTHON_BASE_IMAGE
 # =============================================================================
 
 COPY --from=python_builder /output/ /output
-RUN /output/install-from-bindep && rm -rf /output
+# building EEs require the install-from-bindep script, but not the rest of the /output folder
+RUN /output/install-from-bindep && cd /output && rm -rf !('install-from-bindep')
```

Comment:
I don't think this is valid shell syntax?

But... I think I would prefer to do something like `mv install-from-bindep /usr/local/bin/` and then `rm -rf /output/*`

If you really want to do something like what you have, you could do:

```
$ find /output/* -not -name install-from-bindep -exec rm -rf {} +
``

Reply:
You're right, this doesn't work

> But... I think I would prefer to do something like `mv install-from-bindep /usr/local/bin/` and then `rm -rf /output/*`

But nothing else does it this way. There is value in consistency with other projects, and those invoke /output/install-from-bindep

https://github.com/ansible/ansible-runner/blob/d4a4b26878e43748938d12170f09b1c94c17bad0/Dockerfile#L32

The next thing I was going to look into was the `find` approach, so you may have already gotten working syntax